ProcessLib/HT/HTProcess.cpp

```diff
@@ -64,6 +62,8 @@ void HTProcess::initializeConcreteProcess(
 
     if (_use_monolithic_scheme)
     {
+        checkMPLProperties(mesh, _process_data);
```

Comment:
Why only for monolithic scheme?

Reply:
The staggered scheme uses other governing equations. For instance, the staggered scheme needs the specification of the storage.